.github/workflows/artifact_redirect.yml

```diff
@@ -15,7 +15,3 @@ jobs:
           artifact-path: 0/html/index.html
           circleci-jobs: build_book
           job-title: Click to preview rendered book
-      - name: Check the URL
```

Comment:
Ok so here is the URL checker that You are suggesting that we remove. Is this artifact redirect simply allowing us to click on the build at the bottom and skip that extra step of going to the artifact tab? i suspect that is what it's doing. So you are saying it is finicky?

Could we keep it for now and then if it starts breaking our builds just remove it and have that extra click step from the circle ci build? does that make sense @kysolvik ? i see every build in simpy has a red x which isn't ideal.